lldb-eval/parser_context.h

```diff
@@ -72,6 +72,9 @@ class ParserContext {
   void SetAllowSideEffects(bool allow_side_effects);
   bool AllowSideEffects() const;
 
+  std::tuple<Type::MemberInfo, std::vector<uint32_t>> GetMemberInfo(
+      TypeSP type, const std::string& name);
```

Comment:
Can this be `const`? It looks it only needs `GetEmptyType`, which can be make `const` as well.

Reply:
Done